pronotepy/dataClasses.py

```diff
@@ -1308,41 +1310,21 @@ def __init__(self, client: _ClientBase, json_dict: dict) -> None:
         self.name: Optional[str] = self._resolver(str, "L", strict=False)
         self.date: datetime.date = self._resolver(Util.date_parse, 'Date', 'V')
 
-        self.is_lunch: bool = (self._resolver(int, 'G') == 0)
-        self.is_dinner: bool = (self._resolver(int, 'G') == 1)
+        self.is_lunch: bool = self._resolver(int, 'G') == 0
+        self.is_dinner: bool = self._resolver(int, 'G') == 1
 
-        d_dict = {meal['G']: meal for meal in json_dict["ListePlats"]["V"]}
+        def init_food(d: dict) -> List[Menu.Food]:
+            return [self.Food(client, x) for x in d["ListeAliments"]["V"]]
 
-        self.first_meal: Optional[List[self.Food]] = None
-        self.main_meal: Optional[List[self.Food]] = None
-        self.side_meal: Optional[List[self.Food]] = None
-        self.other_meal: Optional[List[self.Food]] = None
-        self.cheese: Optional[List[self.Food]] = None
-        self.dessert: Optional[List[self.Food]] = None
+        d_dict = {str(meal['G']): meal for meal in json_dict["ListePlats"]["V"]}
+        super().__init__(d_dict)
```

Comment:
I am gonna probably redo this later, right now it works, but it's a little ugly don't you think? I suppose we should add something to the resolver that deals with this. Afaik the same situation is in `Lesson.__init__`.

Reply:
Yes maybe we can try to add a new method in resolver to do that